test/e2e/webhook_test.go

```diff
@@ -25,7 +25,7 @@ import (
 
 	fleetv1alpha1 "go.goms.io/fleet/apis/v1alpha1"
 	"go.goms.io/fleet/pkg/utils"
-	testUtils "go.goms.io/fleet/test/e2e/utils"
+	testutils "go.goms.io/fleet/test/e2e/utils"
```

Comment:
does the golang style say it prefers all lower case for package import name?

Reply:
No, I changed it, throughout the repo we are using lower case for all import aliases noticed this discrepancy and decided to change it